deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -10,209 +10,69 @@
 # non-default instancetype.
 
 # Single-core, Rocket-based recipes
-
-[firesim-singlecore-nic-l2-lbp]
+[firesim-singlecore-sha3-no-nic-llc4mb-ddr3]
 DESIGN=FireSim
-TARGET_CONFIG=L2SingleBank512K_FireSimRocketChipSingleCoreConfig
-PLATFORM_CONFIG=BaseF1Config
-instancetype=z1d.2xlarge
-deploytriplet=None
-
-[firesim-singlecore-no-nic-l2-lbp]
-DESIGN=FireSimNoNIC
-TARGET_CONFIG=L2SingleBank512K_FireSimRocketChipSingleCoreConfig
-PLATFORM_CONFIG=BaseF1Config_F75MHz
+TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimSha3RocketConfig
+PLATFORM_CONFIG=BaseF1Config_F120MHz
```

Comment:
Maybe now is a good time to start prepending, "as opposed to appending", the frequency configs so that we are consistent and don't encourage pitfall-inducing config behavior. @sagark thoughts?

Reply:
yeah, let's do it